src/test/java/blackjack/model/BlackjackGameTest.java

```diff
@@ -0,0 +1,111 @@
+package blackjack.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class BlackjackGameTest {
+    private BlackjackGame blackjackGame;
+    private Dealer dealer;
+    private List<Player> players;
+    private Player firstPlayer, secondPlayer;
+
+    @BeforeEach
+    void init(){
+        blackjackGame = new BlackjackGame();
+        dealer = new Dealer();
+        players = new ArrayList<>();
+        firstPlayer = new Player("first");
+        secondPlayer = new Player("second");
+    }
+
+    @Test
+    void 딜러만_이겼을때(){
+        dealer.getCards().hit(new Card(Denomination.ACE, Suit.DIAMOND));
+        dealer.getCards().hit(new Card(Denomination.TEN, Suit.CLOVER));
```

Comment:
테스트가 많아질수록 세팅하는 부분이 힘들어질 것 같은데, 생성자를 통해 만들면 어떨까
```suggestion
        Dealer dealer = new Dealer(Arrays.asList(new Card(Denomination.ACE, Suit.DIAMOND), new Card(Denomination.TEN, Suit.CLOVER));

```

Reply:
Dealer에 생성자를 추가하여 테스트를 할 때 용이하게 했으나, 현재는 테스트를 위한 코드로만 사용되어 비효율적으로 보여서 추후에 실제 동작하는 코드에서도 해당 생성자를 이용하면 좋을 것 같음

Reply:
++ Player도 동일하게 변경